samples/js/browser/index.html

```diff
@@ -566,6 +566,12 @@ <h3 style="font-weight:500;">Javascript Browser Sample</h3>
             // Multiple languages can be specified for text translation and will be returned in a map.
             if (sdkConfigType == SpeechSDK.SpeechTranslationConfig) {
                 speechConfig.addTargetLanguage(languageTargetOptions.value.split("(")[1].substring(0, 5));
+
+                // If voice output is requested, set the target voice.
+                // If multiple text translations were requested, only the first one added will have audio synthesised for it.
+                if (voiceOutput.checked) {
```

Comment:
Oh boy... we had the checkbox there but it wasn't wired up to do anything? Oops! Thanks for fixing this!